core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -2783,6 +2783,10 @@ public int getSparkEngineBaseExecutorInstances() {
         return Integer.parseInt(getOptional("kylin.engine.base-executor-instance", "5"));
     }
 
+    public String getSparkEngineRequiredTotalCores() {
```

Comment:
If it is possible to merge two configuration `kylin.engine.spark.task-impact-instance-enabled` and `kylin.engine.spark.task-impact-instance-enabled`?